lib/std/fs/wasi.zig

```diff
@@ -9,6 +9,12 @@ const wasi = std.os.wasi;
 const fd_t = wasi.fd_t;
 const prestat_t = wasi.prestat_t;
 
+pub const STDIN_FILENO = 0;
+pub const STDOUT_FILENO = 1;
+pub const STDERR_FILENO = 2;
+
+pub const PATH_MAX = 4096;
```

Comment:
These are libc concepts; they don't really belong here, at least not named like this.

Reply:
Ah, I put these here because the Wasi code that preopens stdin/stdout/stderr is here (below in `preopensAlloc`), so it seemed like a reasonable common place.

I can just duplicate them in std.posix.system and std.c to avoid the fs dependency.

Reply:
Removed these constants from fs/wasi.zig, they're set in c.zig (for wasi-on-libc) and posix.zig (for wasi-without-libc).